public/template.html

```diff
@@ -11,96 +11,175 @@
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap" rel="stylesheet">
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap" rel="stylesheet">
-
-    <style>
-        #EmployerSign {
-            margin-top: 50px;
-            text-decoration: overline;
-            font-family: 'Montserrat', 'sans-serif';
-            font-size: 10px;
-            font-weight: bold;
-        }
-
-        #EmployeeSign {
-            margin-top: 50px;
-            text-decoration: overline;
-            font-family: 'Montserrat', 'sans-serif';
-            font-size: 10px;
-            font-weight: bold;
-        }
-    </style>
 </head>
+<style>
+    #EmployerSign {
+        margin-top: 50px;
+        text-decoration: overline;
+        font-family: 'Montserrat', 'sans-serif';
+        font-size: 10px;
+        font-weight: bold;
+    }
+
```

Comment:
Many of these CSS classes can be replaced with bootstrap helper classes. Try to use bootstrap as much as possible.
https://getbootstrap.com/docs/4.0/utilities/borders/